DS-Session-1/dataSructure/LinkedListImplementationOfQueue.java

```diff
@@ -0,0 +1,108 @@
+package dataSructure;
+
+/**
+ * this class is used to create a node for the linked list
+ * @author Ishita_Mathur
+ *
+ */
+class Node
+{ 
```

Comment:
Class `Node` is almost same for Stack & Queue. A separate class can be used at both implementation.